Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "ft-lockup"
-version = "0.2.0"
+version = "0.3.0"
```

Since it's a breaking change.
```suggestion
version = "1.0.0"
```

---

@evgenykuzyakov , are we following the semver here? 
quoting https://semver.org/, anything can be changed before 1.0.0 release, but I'm okay with moving forward.
```
Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.
```